### DISCUSSION TITLE
Ideas for my project

### DISCUSSION THREAD

**Discussion by Aryan-Falcon** [community/community#131977] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Please give me ideas and software tools that can help me complete  this project: https://aryan-falcon.github.io/nightfalcon/Night-Falcon/
Any help is appreciated.

**Comment by jackabald** [comment]
Tools that will be useful depends on your goals for the application. You can make a beautiful looking website with just HTML, CSS, and JS. Additionally, IMO you should have these skills absolutely perfected before you look into any other tools like react or typescript for enhancing your projects structure and functionality. 

Please, if you like my response, highlight my response as the accepted answer.

**Comment by Aryan-Falcon** [comment]
Do anyone know how I can store things like passwords and usernames securely and have each user have a separate and automatic deployment of the application that is configured to be the same in the beginning? Anything would help.

**Comment by jackabald** [comment, marked as answer]
Firebase, Microsoft Azure, AWS, GCP, etc., are all platforms that will perform User Authentication for you with generally pretty simple ease of use. 

Additionally, you can always build your own backend.
